jetbrains-core/src/software/aws/toolkits/jetbrains/services/codemodernizer/ArtifactHandler.kt

```diff
@@ -85,7 +85,7 @@ class ArtifactHandler(private val project: Project, private val clientAdaptor: G
                 downloadResultsResponse = clientAdaptor.downloadExportResultArchive(job)
             } catch (e: Exception) {
                 CodetransformTelemetry.logApiError(
-                    codeTransformApiNames = CodeTransformApiNames.StartTransformation,
+                    codeTransformApiNames = CodeTransformApiNames.ExportResultArchive,
```

Comment:
The exception thrown in downloadExportResultArchive progress should be under  CodeTransformApiNames.ExportResultArchive.